### PR TITLE
Add size 32 Pix card scheme icon

### PR DESCRIPTION
**Pull request by hris27**
Addresses PMNTSS-1406.

## Purpose

We have to use the size 32 green Pix icon in transactions history instead of the size 24 black one.

## Approach and changes

- Added size 32 Pix card scheme icon.

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

**Review by amelako — APPROVED** [review]

File: packages/icons/web/v2/pix_32.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (connor-baer) [inline packages/icons/web/v2/pix_32.svg:1]: Shouldn't the icon be square?

File: .changeset/sixty-weeks-cry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': patch
+---
+
+Added size 32 Pix card scheme icon
```
Comment (connor-baer) [inline .changeset/sixty-weeks-cry.md:5]:
```suggestion
Added a size 32 Pix card scheme icon.
```

File: packages/icons/web/v2/pix_32.svg (hunk 1 above)
Comment (robinmetral) [reply packages/icons/web/v2/pix_32.svg:1]: No, that's a new icons pack. This is right

**Review by robinmetral — APPROVED** [review]
Looks good judging by the diff and deploy preview 👍 thanks!

**Pull request merged**